test/navi/core_test.clj

```diff
@@ -11,7 +11,10 @@
            [io.swagger.v3.oas.models Operation PathItem]
            [io.swagger.v3.oas.models.media Content StringSchema IntegerSchema JsonSchema
             NumberSchema ObjectSchema ArraySchema MediaType UUIDSchema Schema]
-           [io.swagger.v3.oas.models.parameters Parameter PathParameter HeaderParameter QueryParameter RequestBody]))
+           [io.swagger.v3.oas.models.parameters Parameter PathParameter HeaderParameter QueryParameter RequestBody]
+           [io.swagger.v3.oas.models.responses ApiResponses ApiResponse]))
+
+(set! *warn-on-reflection* true)
```

Comment:
can we move this into the `(comment)` maybe? its useful but as part of a lib it might not be nice to turn this on for everyone importing this.

Reply:
Fixed.